### PR TITLE
AB#9070 Home page translation 

### PR DESCRIPTION
**Pull request by adoprog**

### CODE REVIEW

**Review by chpa7 — COMMENTED** [review]
Looking good, just a few minor points from my end, pending JF's final approval.

And change target branch to develop :)

File: src/items/sxa-content/content/PLAY/playwebsite/Data/ProductCategoriesGrid/Most Popular Product Categories Grid/motorsports.yml [hunk 1]
```diff
@@ -22,3 +22,17 @@ Languages:
...
+    - ID: "26e44e63-d183-4c70-b50d-ad2358a9c1af"
+      Hint: Picture
+      Value: |
+        <image mediaid="{7071C7CE-784D-434B-BBDB-C3552D5CD116}" />
+    - ID: "8cdc337e-a112-42fb-bbb4-4143751e123f"
+      Hint: __Revision
+      Value: "c33431b6-ad23-4e24-98e1-e641d5e5f032"
+    - ID: "ceb0f7a3-e5e1-4e8c-a95a-8bff6f13708f"
+      Hint: Title
+      Value: Sports mécaniques
```
Comment (jeanfrancoislarente) [inline src/items/sxa-content/content/PLAY/playwebsite/Data/ProductCategoriesGrid/Most Popular Product Categories Grid/motorsports.yml:38]: Sports motorisés

File: src/items/sxa-content/content/PLAY/playwebsite/Dictionary/PLAY/book-tickets.yml [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  - ID: "b5e02ad9-d56f-4c41-a065-a133db87bdeb"
+    Hint: __Display name
+    Value: Book Tickets
+  Versions:
+  - Version: 1
+    Fields:
+    - ID: "8cdc337e-a112-42fb-bbb4-4143751e123f"
+      Hint: __Revision
+      Value: "4c8fd6ef-7849-4043-a5c4-04e4d22bd285"
+- Language: "fr-CA"
```
[the remark this answers is not in the document]
Comment (adoprog) [reply src/items/sxa-content/content/PLAY/playwebsite/Dictionary/PLAY/book-tickets.yml]: No need for display name for a dictionary entry.

File: src/items/sxa-content/content/PLAY/playwebsite/Data/CMP Data/Advertisements/PLAY Summit  Urban Hero.yml [hunk 3]
```diff
@@ -35,3 +35,26 @@ Languages:
...
+  Versions:
+  - Version: 1
+    Fields:
+    - ID: "69e8418c-5f11-40a8-9c0d-9e5eacc1229e"
+      Hint: Hero
+      Value: |
+        <image stylelabs-content-id="31904" thumbnailsrc="https://playsummit.sitecoresandbox.cloud/api/gateway/31904/thumbnail" src="https://playsummit.sitecoresandbox.cloud/api/public/content/95619f8c034947a2aa2ce5b39146ccf5?v=5ae3562e" mediaid="" stylelabs-content-type="Image" alt="home-hero-background.jpg" height="850" width="2000" />
+    - ID: "81af116e-235f-44ad-a8e0-794bc3504c7c"
+      Hint: Slogan
+      Value: "PRÊT | STABLE | JOUEZ !"
```
Comment (jeanfrancoislarente) [inline src/items/sxa-content/content/PLAY/playwebsite/Data/CMP Data/Advertisements/PLAY Summit  Urban Hero.yml:48]: PRÊT | PARTEZ | JOUEZ !

Sorry, missed this one - should be it!

Comment (adoprog) [reply src/items/sxa-content/content/PLAY/playwebsite/Data/CMP Data/Advertisements/PLAY Summit  Urban Hero.yml]: Updated

**Review by jeanfrancoislarente — APPROVED** [review]

**Pull request merged**